RIOT-OS/RIOT
# pkg/c25519: cleanup in Makefiles

### Contribution description

While running `make distclean`, I noticed that `tests/pkg_c25519` was raising a warning because of `test` target being redefined.

This PR is removing this target.

I also noticed that the package distclean target is using `rm -rf $(PKG_SHAFILE)` with `$(PKG_SHAFILE)` not defined anywhere. This result in a bare call to `rm -rf` which I find not nice. So this is also removing this (in a separate commit). I can drop this from this PR if there's a good reason to keep it.




### Testing procedure

A green Murdock should be ok. `make -C tests/pkg_c25519 distclean` should not raise any warning. No call to `rm -rf ` without parameter is done.




### Issues/PRs references

None
